.github/workflows/pypi-release.yml

```diff
@@ -0,0 +1,25 @@
+name: PyPI release
+on: [push]
+
+jobs:
+  pypi_release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.x
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.x
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade setuptools wheel twine
+    - name: Build
+      run: |
+        python setup.py sdist bdist_wheel
+        twine check dist/*
+    - name: Publish package
+      if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags')
```

Comment:
This is the line I disabled for doing the test deployment

Reply:
We are still the ones dealing with the versions via the `setup.py`
https://github.com/kivy/kivy-ios/blob/56431b6/setup.py#L19
Once the tag is pushed the only thing this action is doing is doing creating the archive using the `setup.py` and publishing it